runtime/compiler/env/JITServerPersistentCHTable.cpp

```diff
@@ -31,14 +31,23 @@
 
 
 JITServerPersistentCHTable::JITServerPersistentCHTable(TR_PersistentMemory *trMemory)
-   : TR_PersistentCHTable(trMemory)
+   : TR_PersistentCHTable(trMemory),
+   _classMap(decltype(_classMap)::allocator_type(TR::Compiler->persistentAllocator()))
    {
+   _chTableMonitor = TR::Monitor::create("JIT-JITServerCHTableMonitor");
```

Comment:
I wonder I should we do if the monitor cannot be initialized. Maybe throw bad_alloc as if we couldn't allocate the memory.

Reply:
Probably.